debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -596,6 +597,26 @@ public static AutoCreateMode parse(String value, String defaultValue) {
                             "the current filter configuration (see table/database include/exclude list properties). If the publication already" +
                             " exists, it will be used. i.e CREATE PUBLICATION <publication_name> FOR TABLE <tbl1, tbl2, etc>");
 
+    public static final Field REPLICA_IDENTITY_AUTOSET_VALUES = Field.create("replica.identity.autoset.values")
```

Comment:
Would it be possible to introduce an `enum` that will represent allowd values? In case of `INDEX` the index name would be provided via a method.
Or `io.debezium.connector.postgresql.connection.ServerInfo.ReplicaIdentity` should be re-used - maybe extract into a standalone class.

Reply:
>Would it be possible to introduce an enum that will represent allowd values?

@jpechane Keep in mind that this property still needs to be a `String` as users will need to set up `REPLICA IDENTITY` for each table captured (at least for a subset of these tables).

Reply:
ok, makes sense. Just please use comma as separator, not semicolon to make it consistent with `table.include.list` etc.